python/py-termstyle/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-termstyle
+version             0.1.11
+revision            0
+
+categories          python
+supported_archs     noarch
+license             BSD
+maintainers         nomaintainer
+
+description         console colouring for python
+long_description    {*}${description}
+homepage            https://github.com/timbertson/termstyle
+
+checksums           rmd160  6525c56a3c3b57bf3fd7af4220c0de416948fc08 \
+                    sha256  ef74b83698ea014112040cf32b1a093c1ab3d91c4dd18ecc03ec178fd99c9f9f \
+                    size    4580
+
+python.versions     37 38 39 310
+
+
+if {${name} ne ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-setuptools
```

Comment:
why is this also a runtime dependency?

Reply:
The author specified this dependency in install_requires in setup.py. You are right, it is used only for setup.py.